### DISCUSSION TITLE
Collaborator's Doubts?🧑‍💻

### DISCUSSION THREAD

**Discussion by LOGESH-B** [gbowne1/codebooker#88] [Q&A] [answered]
Hii @gbowne1 
As I am a collaborator is it right to work  on a issue by creating and self assigning the issue or I directly work on that without ceating a issue?

**Comment by gbowne1** [comment, marked as answer]
I would prefer issues created.  This way it can be tracked.

My thoughts is that anyone can and should create issues regardless of who works on them.  Issues can even be assigned to multiple people.  Say an issue requires both backend and frontend work. A person can work on the frontend and another person can work on backend of the issue/task.

Btw, if you want to add issue tasks, you can use a dash and two brackets like - [ ] taskhere to create issue tasks which can have a x placed in between the brackets to show a completed task.

But after the issue is created, anyone could assign themselves and that is ok.  Several of my projects have collaborators that work this way.  Just let us know by saying 'I'm working on this'

**Comment by gbowne1** [comment]
Collaborators should also use the Project board in the Projects tab.  I haven't spent a lot of time organizing it, but it allows collaborators, maintainers, etc to see all issues, assignments at a glance.

**Reply by gbowne1** [reply]
At that point it becomes increasingly necessary to use the right hand sidebar in both issues and PRs.  GitHub's built in automation will be able to close issues once completed as well as other things.

**Reply by gbowne1** [reply]
I also prefer TODO labeled issues instead of TODO files especially if you don't use VSCode or Visual Studio or an IDE that uses extensions or plugins.  We do have a TODO.md file but I found it hard to keep up with keep it updated.  Some people like to //TODO comment in their code for unfinished things. This allows people to see what features and code needs done/completed.

**Reply by gbowne1** [reply]
I also encourage people working on the project to post errors, warnings, bugs they find in the browser and also test by opening the localhost port on different devices and browsers

And especially posting screenshots of the browser and console.
